src/test/java/echoServer/EchoServerTest.java

```diff
@@ -60,6 +100,15 @@ public String readLine() {
         public void writeLine(String message) {
             written.add(message);
         }
+
+        @Override
+        public void close() {
+            events.add("Closed ReadableWriteable IO streams");
+        }
+
+        public TestReaderWriter closeConnection() {
```

Comment:
**question**: I wonder if something like `disconnect` might be more clear, especially now that `close` is part of the API. What do you think?

Reply:
I think `disconnect` is super clear - thanks for the suggestion!